cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -711,7 +711,12 @@ def make_resource(self, data, **kwargs):
 class IamSchema(BaseSchema):
     """Represent the schema of IAM for HeadNode and Queue."""
 
-    instance_role = fields.Str(metadata={"update_policy": UpdatePolicy.SUPPORTED})
+    instance_role = fields.Str(
+        metadata={"update_policy": UpdatePolicy.SUPPORTED}, validate=validate.Regexp("^arn:.*:role/")
```

Comment:
let's add unit tests for them, see `cli/tests/pcluster/schemas/test_schema_validators.py`

Reply:
Added